content/coronavirus_landing_page.yml

```diff
@@ -209,3 +209,15 @@ content:
     date: 1st April 2020
     time: 5:00pm
     show_video: false
+  # https://schema.org/SpecialAnnouncement fields
+  special_announcement_schema:
+    category: https://www.wikidata.org/wiki/Q81068910
```

Comment:
Is this something that will change?

Reply:
Not for this pandemic 😬

Reply:
But we might want to switch to wikipedia or a different canonical page based on advice